contracts/cw-proposal-multiple/src/contract.rs

```diff
@@ -335,6 +339,7 @@ pub fn execute_execute(
         VoteResult::SingleWinner(winning_choice) => {
             let response = match winning_choice.msgs {
                 Some(msgs) => {
+                    let res = Response::default();
```

Comment:
nit: imo code is a little easier to read if we remove this and just construct a `Response::default()` when returning. Otherwise, the reader has to read up and see what else has been added.